tools/bigquery-query-plan-exporter/bqutil/bqutil.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from google.cloud import bigquery
+from datetime import datetime, timedelta
+import sys
+import json
+
+
+class BQUtil(object):
+    def __init__(self, project=None, location='US'):
+        self.project = project
+        self.location = location
+        self.bq = None
+
+    def get_client(self):
+        if self.bq is None:
+            self.bq = bigquery.Client(project=self.project, location=self.location)
```

Comment:
\> 80 chars. split into two lines. see go/python-style and run linter to catch style things I missed.